Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingTreeMac.mm

```diff
@@ -90,6 +90,13 @@
     });
 }
 
+void RemoteScrollingTreeMac::clearNodesWithUserScrollInProgress()
+{
+    RunLoop::main().dispatch([protectedThis = Ref { *this }, this] {
+        RemoteScrollingTree::clearNodesWithUserScrollInProgress();
```

Comment:
hmm.. `RemoteScrollingTree::clearNodesWithUserScrollInProgress()` is not a static function, can we call it like this?

Reply:
yes; it's calling the super method

Reply:
oh! My bad